README.md

```diff
@@ -20,7 +20,7 @@ This extension adds a `budgetBreakdown` property to the `planning` section of OC
 
 This extension also extends the `budget` block with the following additional properties for use in the `budgetBreakdown` section:
 
-* `budget.sourceParty` - an organization reference, linking to the entry in the `parties` section describing the organization providing the funds for this part of the budget
+* `budget.sourceParty` - an organization reference, linking to the entry in the `parties` section describing the organization providing the funds for this part of the budget. This party is ought to have the `funder` role
```

Comment:
> The party must have the 'funder' role

Change to MUST (we can do that in community extensions).

Use single quotes for codes (backticks for fields). Tests failed because of this.